### PR TITLE
bump version to 1.18.3

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "1.18.1" %}
+{% set version = "1.18.3" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should we be doing 1.18.2 first?

**Comment by mrocklin** [comment]
I screwed up and failed to include the updated tornado version requirement
in requirements.txt, hence the version number skip.  I would be very happy
to learn of a better way to handle a PyPI upload mistake.

**Comment by jakirkham** [comment]
Oh ok. Unfortunately PyPI makes it basically impossible to change things after they are released. So doing a minor version to straighten things out is the right strategy.

LGTM.

**Comment by mrocklin** [comment]
In general I've been fairly casual/sloppy about packaging Dask (this release itself is the result of missing a couple of small things).  At some point we should probably enforce more mature practices.

**Pull request merged**